### DISCUSSION TITLE
Blocks found but no credit

### DISCUSSION THREAD

**Discussion by JimmyBoy86** [Chia-Network/chia-blockchain#8013] [Support]
In the Space Pool app, it shows me a block it has found. However, I was not paid out 0.25 coins.
Is there a mistake here or am I wrong?

**Comment by keliew** [comment]
What would Space Pool say?

What does `chia wallet show` say? And `chia wallet get_transactions`.

**Reply by JimmyBoy86** [reply]
SpacePool says 1 block was found but no 0.25 was credited
No 0.25 is listed in the wallet or in the transactions.
